.golangci.yaml

```diff
@@ -44,6 +45,192 @@ issues:
+        alias: clientmapbuilder
+      - pkg: github.com/gardener/gardener/pkg/client/kubernetes/clientmap/fake
+        alias: fakeclientmap
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/clientset/versioned
+        alias: seedmanagementclientset
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/informers/externalversions
+        alias: seedmanagementinformers
+      - pkg: github.com/gardener/gardener/pkg/client/seedmanagement/clientset/versioned/fake
+        alias: fakeseedmanagement
+      - pkg: github.com/gardener/gardener/pkg/client/settings/clientset/versioned
+        alias: settingsclientset
+      - pkg: github.com/gardener/gardener/pkg/client/settings/informers/externalversions
+        alias: settingsinformers
+      - pkg: github.com/gardener/gardener/pkg/client/settings/listers/settings/v1alpha1
+        alias: settingsv1alpha1lister
+      - pkg: github.com/gardener/gardener/pkg/controllerutils/(\w+)
+        alias: ${1}utils
+      - pkg: github.com/gardener/gardener/pkg/envtest
+        alias: gardenerenvtest
+      - pkg: github.com/gardener/gardener/pkg/extensions
+        alias: gardenerextensions
+      - pkg: github.com/gardener/gardener/pkg/gardenlet/bootstrap/util
+        alias: gardenletbootstraputil
+      - pkg: github.com/gardener/gardener/pkg/healthz
+        alias: gardenerhealthz
+      - pkg: github.com/gardener/gardener/pkg/mock/apiserver/authorization/authorizer
+        alias: mockauthorizer
+      - pkg: github.com/gardener/gardener/pkg/mock/\w+/(\w+)
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/core/v1
+        alias: mockcorev1
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/tools/(\w+)
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/mock/client-go/util/(\w+)
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/operation/.*/(\w+)/mock$
+        alias: mock${1}
+      - pkg: github.com/gardener/gardener/pkg/operation/garden
+        alias: gardenpkg
+      - pkg: github.com/gardener/gardener/pkg/operation/seed
+        alias: seedpkg
+      - pkg: github.com/gardener/gardener/pkg/operation/shoot
+        alias: shootpkg
+      - pkg: github.com/gardener/gardener/pkg/operator/client
+        alias: operatorclient
+      - pkg: github.com/gardener/gardener/pkg/registry/(\w+)/rest
+        alias: ${1}rest
+      - pkg: github.com/gardener/gardener/pkg/registry/\w+/(\w+)/storage$
+        alias: ${1}store
+      - pkg: github.com/gardener/gardener/pkg/registry/\w+/([^r]\w+)
+        alias: ${1}registry
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/client
+        alias: resourcemanagerclient
+      - pkg: github.com/gardener/gardener/pkg/resourcemanager/predicate
+        alias: resourcemanagerpredicate
+      - pkg: github.com/gardener/gardener/pkg/utils/(\w+)
+        alias: ${1}utils
```

Comment:
I would like to keep `kutil` and `gutil` for `utils/kubernetes` and `utils/gardener`

Reply:
Can we not keep it, `kubernetesutils` and `gardenerutils`, if not I will have to give names statically for all packages in utils and can't use regex, cause go regex doesn't allow negative lookahead(i.e. It can't ignore negation of string in regex).
In general the more regex we will use, it will be easy to maintain things.

Reply:
Yeah, true. Well, then let's keep it like this for now and see whether we want to change it in the future (at the price of defining the aliases explicitly.